Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -2440,50 +2440,39 @@ const AtomString& RenderStyle::hyphenString() const
 
 void RenderStyle::adjustAnimations()
 {
-    if (m_nonInheritedData->miscData->animations.isNone())
+    if (animations().isNone())
         return;
 
-    auto& animationList = ensureAnimations();
+    auto& list = ensureAnimations();
 
-    // Get rid of empty animations and anything beyond them
-    for (size_t i = 0, size = animationList.size(); i < size; ++i) {
-        if (animationList[i].isEmpty()) {
-            animationList.resize(i);
-            break;
-        }
-    }
+    list.prepareForUse();
```

Comment:
I’m a little surprised we are using a local variable here.
```suggestion
    ensureAnimations().prepareForUse();
```
Same question about the other 3 functions.